app/models/activity.rb

```diff
@@ -7,6 +7,8 @@ class Activity < ActiveRecord::Base
 
   validates :content, :title, :team, presence: { if: ->(act) { act.kind == 'status_update' } }
 
+  delegate :students, to: :team
```

Comment:
Ideally, there would be a unit-test for this. It's a simple one, really, just to avoid regressions. But if you feel coping with rspec is too much at this time, we can skip that (or leave that for later)